.github/workflows/rust.yml

```diff
@@ -57,12 +57,14 @@ jobs:
           cargo test -p dicom-pixeldata
           cargo check -p dicom
         env:
-          RUSTFLAGS: -W warnings
+          RUSTFLAGS: -A warnings # allows warnings, to not pollute CI with warnings that are no longer valid with stable rust version
       # run Clippy with stable toolchain
       - if: matrix.rust == 'stable'
-        run: cargo clippy
+        run: |
+          cargo clean # Cleaning cache, because sometimes clippy shows outdated warnings from cache
```

Comment:
I don't think that this was an outdated warning. Can we try #611 first?

Reply:
I assumed it was a Clippy cache issue because the error didn't occur locally – however, I must have been running something incorrectly, as the rebase resolved the problem.